### PR TITLE
Db layer

### PR DESCRIPTION
**Pull request by pizzaeueu**

### CODE REVIEW

File: src/main/java/com/inquirer/dao/QuestionDaoSQL.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
+package com.inquirer.dao;
```
Comment (mafitov) [inline src/main/java/com/inquirer/dao/QuestionDaoSQL.java]: Rename class QuestionDaoImpl

File: src/main/java/com/inquirer/dao/QuestionDaoSQL.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+    }
+
+    public List<Answer> getAnswersForQuestion(Question question) throws SQLException {
+        List<Answer> answers =  new ArrayList();
+
+        try (Connection con = DriverManager.getConnection(url,username,password);
+             PreparedStatement statement = con.prepareStatement(SELECT_ANSWER_FOR_QUESTION_QUERY);
+             ) {
+            try {
```
Comment (mafitov) [inline src/main/java/com/inquirer/dao/QuestionDaoSQL.java]: Remove double try, seems no need here

File: src/main/java/com/inquirer/dao/QuestionDaoSQL.java [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+             ResultSet resultSet = statement.executeQuery()) {
+
+            while (resultSet.next()) {
+                Question tempQuestion = new Question();
+                tempQuestion.setId(resultSet.getInt("id"));
+                tempQuestion.setTitle(resultSet.getString("title"));
+                questions.add(tempQuestion);
+            }
+        } catch (SQLException e){
+            e.printStackTrace();
```
Comment (mafitov) [inline src/main/java/com/inquirer/dao/QuestionDaoSQL.java]: Add logger ERROR instead e.printStackTrace() everywhere in the project

File: src/database/inquirer_database_script.sql [hunk 4]
```diff
@@ -0,0 +1,54 @@
+/*
```
Comment (mafitov) [inline src/database/inquirer_database_script.sql]: Move database/inquirer_database_script.sql to root folder. src folder should contain only main and test folders according maven specification

File: src/main/java/com/inquirer/dao/QuestionDaoSQL.java [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+            DriverManager.registerDriver(driver);
+        } catch (SQLException e){
+            e.printStackTrace();
+        }
+    }
+
+    public void addUser(User user) throws SQLException {
+
+        try (Connection con = DriverManager.getConnection(url,username,password);
+            PreparedStatement statement = con.prepareStatement(ADD_USER_QUERY)) {
```
Comment (viacheslav-bogdanov) [inline src/main/java/com/inquirer/dao/QuestionDaoSQL.java]: Extract this 2 line in separate method. You repeat them every time with different queries for statement

**Comment by mafitov** [comment]
:+1:

**Pull request merged**